Gopkg.toml

```diff
@@ -24,11 +24,24 @@ required = ["github.com/kubernetes/repo-infra/verify/boilerplate/test"]
 
 [[override]]
   name="k8s.io/apimachinery"
-  version="kubernetes-1.10.1"
+  version="kubernetes-1.12.4"
 
 [[override]]
   name="k8s.io/client-go"
-  version="kubernetes-1.10.1"
+  version="kubernetes-1.12.4"
+
+[[override]]
```

Comment:
A question here: Why did you add another three overrides here?

Reply:
If we just upgrade the client-go, this three older repo will not work, they should keep the same version :)

Reply:
Got it, thanks @fisherxu :-)